portfolio/src/main/webapp/index.html

```diff
@@ -4,15 +4,42 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="headerContent">
+      <div id="imageContainer">
+	<img src="images/mainSelfie.jpg"  alt="Picture of me" id="headerSelfie">
+      	<button  id="cycleButton" onclick="cyclePictures()"> Cycle Pictures </button>
+      </div>	
+      <div id="greetingContainer">
+      	<h1>Welcom to my portfolio</h1>
+      	<h3>My name is Malachi Brewer. I am a sophomore at the University of Georgia.</h3>
+	<h3>Welcome Buds! I created this site to house random pieces of information about me for anyone who might 
+           be curious. Clicking on the buttons below will take you to different parts of the page.</h3> 
+	<ul id="headerNav"> 
+	 <li><a href="#aboutMe">About Me</a></li>
+ 	 <li><a href="#education">Education</a></li>
+	 <li><a href="#professional">Professional</a></li>
+	 <li><a href="#randomFacts">Random Facts</a></li>
+	</ul>
+      </div>
     </div>
+    <h1 id="aboutMe"> About Me </h1> 
```

Comment:
Let's use lowercase and dash delimited ids and class names (i.e. "about-me") here and throughout the file. See here for more info: go/htmlcssstyle#ID_and_Class_Name_Delimiters